x-pack/platform/plugins/shared/fleet/public/hooks/use_input.ts

```diff
@@ -173,6 +173,7 @@ export function useRadioInput(defaultValue: string, disabled = false) {
     setValue,
     value,
     hasChanged,
+    validate: () => true,
```

Comment:
why is this needed?

Reply:
This was needed because of the linter failing on [this line](https://github.com/criamico/kibana/blob/fcc973ce9a5c2d563ad10966e7f9710f3ef4de6b/x-pack/platform/plugins/shared/fleet/public/applications/fleet/components/fleet_server_instructions/hooks/use_fleet_server_host.ts#L123):
The inputs on `[validateInputs](https://github.com/criamico/kibana/blob/fcc973ce9a5c2d563ad10966e7f9710f3ef4de6b/x-pack/platform/plugins/shared/fleet/public/hooks/use_input.ts#L12-L22)` now need to accept an additional `useSwitchInput` that wasn't there before and didn't respect this signature

```
export interface FormInput {
  validate: () => boolean;
}
```

I tried to make `validate` optional in `FormInput` but it was still failing.